### ISSUE TITLE
Can't re-create previously created environment

### ISSUE THREAD

**Issue by williamokano-dh** [litmuschaos/litmus#4339]
**What happened**:
Got the following error when attempting to create a new environment with the same name from a previously deleted environment.
```
write exception: write errors: [E11000 duplicate key error collection: litmus.environment index: environment_id_1 dup key: { environment_id: "testenv" }]
```

**What you expected to happen**:
Be able to create the new environment. As it is not shown as disabled but rather it's removed from the listing, and the environment ID is automatically generated based on the environment name, the server throws an error complaining about duplicated id.

**Where can this issue be corrected? (optional)**

**How to reproduce it (as minimally and precisely as possible)**:
1. Create an environment and give it a name, let's say `test-environment-name-example`
2. Remove the environment
3. Create an environment and give it the very same name `test-environment-name-example`

**Anything else we need to know?**:

**Comment by williamokano-dh** [comment]
Hi @Saranya-jena , any updates in this matter?

**Comment by Saranya-jena** [comment]
Hi @williamokano-dh , we are working on this issue, since this requires thorough testing, it will be available in 3.5.0 release.

**Comment by williamokano-dh** [comment]
Thanks for the update @Saranya-jena
